src/appshell/appshell.cpp

```diff
@@ -83,6 +83,15 @@ int AppShell::run(int argc, char** argv)
 
     QCoreApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
 
+    //! NOTE: For unknown reasons, Linux scaling for 1 is defined as 1.003 in fractional scaling.
+    //!       Because of this, some elements are drawn with a shift on the score.
+    //!       Let's make a Linux hack and round values above 0.75(see RoundPreferFloor)
+#ifdef Q_OS_LINUX
+    QGuiApplication::setHighDpiScaleFactorRoundingPolicy(Qt::HighDpiScaleFactorRoundingPolicy::RoundPreferFloor);
```

Comment:
@Eism please provide more information. this change breaks fractional scale values. if you have a particular issue with one DE and display server combination, it is best to have it corrected upstream and not break this for everyone else.

with the current code we get:
1X scaling for 0.75-1.749999
2X scaling for 1.75-2.749999
3X scaling for 2.75-3.749999
etc

this is not a good system. using passthrough would be much better.